xhyve/xhyve.go

```diff
@@ -610,6 +613,49 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
+func readLine(path string) []string {
+	lines := []string{}
+	inFile, err := os.Open(path)
+	if err != nil {
+		log.Debugf("Not able to open %s", path)
+	}
+	defer inFile.Close()
+	scanner := bufio.NewScanner(inFile)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	return lines
+}
+
+func extractKernelOptions(d *Driver) error {
+	log.Debugf("Extracting Kernel Options")
+	volumeRootDir := d.ResolveStorePath(isoMountPath)
+	readIsoLinuxCfg := []string{}
+	if d.BootCmd == "" {
+		err := filepath.Walk(volumeRootDir, func(path string, f os.FileInfo, err error) error {
+			if strings.Contains(path, "isolinux.cfg") {
+				readIsoLinuxCfg = readLine(path)
+			}
+			return nil
+		})
+		if err != nil {
+			return err
+		}
+		for _, line := range readIsoLinuxCfg {
```

Comment:
@zchee yes, I will add this logic to `readLine` function and then return only matched line.